CITATION.cff

```diff
@@ -0,0 +1,41 @@
+# YAML 1.2
+---
+authors:
+cff-version: "1.1.0"
```

Comment:
Maybe using the latest spec version (1.2.0 I believe) would be better?

Reply:
done, please check !!!